### PR TITLE
Added conda.recipe

### PR DESCRIPTION
**Pull request by Darknez07**
Fixes #

## Please describe what changes you made, in as much detail as possible
 On the basis of issue #776 
I am trying to build the conda package
Might need a new set of eyes for the yaml file
Please provide stern review on the code
mindsdb

### CODE REVIEW

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/> 
   - - -
   ***I have read the CLA Document and I hereby sign the CLA***
   - - - 
   **Darknez07** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You can retrigger this bot by commenting **recheckcla** in this Pull Request</sub>

**Comment by Darknez07** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Darknez07** [comment]
recheckcla

**Comment by Darknez07** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Darknez07** [comment]
**recheckcla**

File: conda.recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        - python
+        - pip
+    build:
+        - yaml
+    run:
+        - python>=3.5
+
+dependencies:
+        - pip:
+            - mindsdb_native == 2.13.0
```
Comment (torrmal) [inline conda.recipe/meta.yaml:34]: Thank you. @Darknez07 can this infor be pulled from the requirements.txt so we don't need to manage two separate ones?
@George3d6 thoughts?

Comment (Darknez07) [reply conda.recipe/meta.yaml:34]: yes it can be we can use jinja

Comment (Darknez07) [reply conda.recipe/meta.yaml:34]: even versions can be pulled

**Comment by George3d6** [comment]
@Darknez07 Looks good, tests are failing for unrelated reasons (will be fixed soon). We need to test this internally and add it to the docs, but what you've provided thus far looks like a very promising starting point, we can tweak it to better suit our use case later.

Next up @ZoranPandovski we should try integrating this in our docs.

Thanks a million for your help :)

**Pull request merged**